.github/workflows/build_scoped_rector.yaml

```diff
@@ -40,7 +40,9 @@ jobs:
             -  run: rm -rf rector-build/packages-tests rector-build/rules-tests rector-build/tests
 
             # 2. downgrade rector
-            -   run: sh build/downgrade-rector.sh rector-build
+            -   run: |
+                    php -d memory_limit=-1 bin/rector process bin rector.php --config build/config/config-downgrade.php --working-dir rector-build --ansi
```

Comment:
This should be in `build/downgrade-rector.sh`. Why is separated?

Reply:
in the downgrade-rector.sh, the `$BUILD_DIRECTORY` seems got per-directory, so use manual dir definition here.

Reply:
I see. Could you update the script to run on all directories at once instead per-directory?

Reply:
I will try